msgraph-sdk-raptor-compiler-lib/MicrosoftGraphTypescriptCompiler.cs

```
@@ -67,24 +67,37 @@ public CompilationResultsModel CompileSnippet(string codeSnippet, Versions versi
             using var outputWaitHandle = new AutoResetEvent(false);
             using var errorWaitHandle = new AutoResetEvent(false);
             tscProcess.OutputDataReceived += (sender, e) => {
-                if (string.IsNullOrEmpty(e.Data))
+                try
                 {
-                    outputWaitHandle.Set();
+                    if (string.IsNullOrEmpty(e.Data))
+                    {
+                        outputWaitHandle.Set();
+                    }
+                    else
+                    {
+                        stdOuputSB.Append(e.Data);
+                    }
                 }
-                else
+                catch(Exception ex)
```

error swallowing, remove or log